distribution/docs/src/main/resources/_contents/_formats/supported-file-formats-contents.adoc

```diff
@@ -0,0 +1,335 @@
+
+====== NITF-specific Attributes Provided by ${cal-branding}
+
+${cal-branding} includes support for accessing additional metadata from specific file formats, primarily NITF. In addition to the above attributes, ${branding} can extract these attributes from NITF resources, if populated:
+
+.${cal-branding} Additional NITF Attributes
+
+${branding} has the capability to extract many NITF-specific Attributes from NITF files. (available if populated)
+
+* General Attributes
```

Comment:
They are included in the list of attributes common to all file types. As this is the second time I've gotten this comment, I'm going to include them in this list as well.